deploy/internal/configmap-postgres-initdb.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: noobaa-postgres-initdb-sh
+  labels:
+    app: noobaa
+data:
+  initdb.sh: |
```

Comment:
Can we divide this into a script and not write the whole script in the yaml?

Reply:
@liranmauda , the idea is to inject this script into the existing Postgres image. WDYT?